### DISCUSSION TITLE
seems core Markdown does this too?

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card#24] [Q&A] [answered]
HI, 

Please let me ask below, as I am confused a bit as to the why for this card. What you have in your example is like a few cards I use myself, so Ive tested:

```
type: markdown
content: >-

  Sun state: <b>{{ states('sun.sun') }}</b>, elevation: {{
  state_attr('sun.sun','elevation') }}</br>

  <b>Hello</b> there!<p>General <u>Kenobi!</u></p>

  <img src="https://i.redd.it/ltxppihy4cyy.jpg" width="70%"/></br>

  <ha-icon icon="mdi:speaker"></ha-icon> Volume: {{
  states('input_number.radio_volume') }}%</br>

  <center><img
  src="https://vignette.wikia.nocookie.net/starwars/images/f/fa/Modal_Nodes_02.jpg"
  width={{ states('input_number.radio_volume')|float *100 }}%/></center>
```

and it seems identical to what you are doing. Note I did change the template in the slider a bit but other than that, seems perfectly fine and identical

What is it I'm  not seeing in Markdown that makes your new card fill the gap?

No criticism, always cool when people create nice extra cards. Just like to know what I am not seeing.. ;-0

thanks!

![Nov-07-2023 11-37-50](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-HTML-Jinja2-Template-card/assets/33354141/8f3b5b07-b2eb-47b6-8f1d-7575f3f0d90b)

**Comment by PiotrMachowski** [comment, marked as answer]
Well, at the time of creation of this card Markdown card didn't support the Jinja templates. At this moment most cases can be handled by the Markdown card, but there are some things it can't do.

**Reply by Mariusthvdb** [reply]
right, I didnt realize this custom card was already 4 years old... sorry bout that

**Reply by PiotrMachowski** [reply]
![old.gif](https://media.tenor.com/9QX1-Pf4AowAAAAC/aging-getting-old.gif)
